src/plugins/mf_priority.cpp

```diff
@@ -98,11 +126,27 @@ int64_t priority_calculation (flux_plugin_t *p,
         return -1;
     }
 
+    auto it = released_jobs.find (jobid);
+    if (it != released_jobs.end ()) {
+        // job has an "age" associated with it; fetch the timestamp
+        t_released = it->second;
+        // get current time
+        t_current = std::chrono::duration_cast <std::chrono::duration<double>>
+            (std::chrono::system_clock::now ().time_since_epoch ()).count ();
+        // calculate age of job
+        age_factor = t_current - t_released;
+    } else {
+        // job does not have an age; don't factor in the age in
+        // the priority calculation
+        age_factor = 0;
+    }
```

Comment:
I'd suggest refactoring this find into a function, something like `double released_job_age (jobid)`.

Reply:
Ideally `chrono::duration<double>` as the return there rather than a regular double unless you have a reason you need it as a double in all cases.  I'll comment a couple other places, but try to avoid using `.count()` unless you're interfacing with C code or really, really need a bare number.  Chrono prevents a lot of mistakes when you use its types, and it's a bit of a shame to immediately convert that away.